source/PlayerInfo.cpp

```diff
@@ -2016,6 +2018,58 @@ void PlayerInfo::SetTravelDestination(const Planet *planet)
 
 
 
+void PlayerInfo::QueueTeleport(const Planet *teleportPlanet, bool _flagshiponly)
+{
+	teleportDestination = teleportPlanet;
+	flagshiponly = _flagshiponly;
```

Comment:
I'll be honest using an underscore for the function argument feels weird
You should probably rename it to smth like isFlagshipOnly and newTeleportPlanet

Reply:
done